docs/USER_EXPERIENCE_DESIGN.md

```diff
@@ -87,6 +89,20 @@ Examples:
 
 The following guidelines help to uphold the above principles.
 
+### Defaults
+
+#### Don't lose data
+
+When presented with the choice to trade data loss for another benefit, like
+performance, Vector should retain data by default. Data loss is unacceptable
```

Comment:
It may be worth mentioning the _resource_ usage aspects of this as well - buffering events locally, will be more resource intensive, for example